occurrences/admin.py

```diff
@@ -312,7 +315,7 @@ def get_contact_email(self, obj):
 
 
 @admin.register(EnrolleePersonalData)
-class EnrolleePersonalDataAdmin(admin.ModelAdmin):
+class EnrolleePersonalDataAdmin(AuditlogAdminViewAccessLogMixin, admin.ModelAdmin):
```

Comment:
As EnrolleePersonalDataAdmin's list_display contains name, phone_number and email_address which are GDPR data, wouldn't it make sense to log the views because of this?
```suggestion
class EnrolleePersonalDataAdmin(AuditlogAdminViewAccessLogMixin, admin.ModelAdmin):
    enable_list_view_audit_logging = True
```

Reply:
Yea, this could be added, since there are so many GDPR columns.